scripts/cc-cli/src/commands/collectCoins.ts

```diff
@@ -20,7 +20,17 @@ export function makeCollectCoinsCmd() {
     .description("Swap GCRE for CTC")
     .addOption(externalAddressOpt)
     .addOption(burnTxHashOpt)
-    .action(collectCoinsAction);
+    .action(collectCoinsActionSync);
```

Comment:
Is this an intended change? Looks like it was meant only for debugging purposes ?

Reply:
With the previous async function the process would hang after even after completing the collect coins validation. For the CLI only that isn't a major issue because you can just `ctrl-c` out of it but the `execa` command runner blocks until the child process exits. Rewriting it in this synchronous fashion made it a little bit easier to ensure we got correct and easily testable behavior